src/Microsoft.Azure.SignalR/ServiceConnectionScope.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.SignalR
+{
+    /// <summary>
+    /// When using Azure SignalR Service ServiceConnectionScope class tries to ensure that all the messages being sent to clients 
+    /// within its scope are using the same connection from App Server to the Service thus helping to ensure the order of messages.
+    /// </summary>
+    public class ServiceConnectionScope : IDisposable
```

Comment:
We expect people to use this in their hubs? I think that's a non starter and it ties your Hubs to the service logic.